### PR TITLE
Add update gofmt script

### PR DESCRIPTION
**Pull request by m1093782566**
Adding update-gofmt script.

### CODE REVIEW

**Comment by m1093782566** [comment]
@ICGog @deepak-vij @shivramsrivastava

**Comment by shivramsrivastava** [comment]
@m1093782566 Thanks for this PR.
Do we need a separate PR for the changes in PR [#27](https://github.com/kubernetes-sigs/poseidon/pull/27), can't it be merged in this same PR itself?

**Comment by m1093782566** [comment]
@shivramsrivastava

This PR is related to gofmt and #27 is about golint. I would like to keep them seperated.

**Comment by m1093782566** [comment]
@shivramsrivastava 

The CI job `pull-poseidon-verify` should not pass for now, right?

Is it a blocking issue that prevents this PR getting merged?

**Comment by shivramsrivastava** [comment]
@m1093782566 its a bug in the verify script.
The script it trying to use a missing command  'bc'.
I have rasied a PR #30 , will do a reset after that is merged.

**Comment by m1093782566** [comment]
@shivramsrivastava 

PR changed, would you please take a look again and LGTM it if you are happy with it :)

File: hack/update-gofmt.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors.
```
Comment (shashidharatd) [inline hack/update-gofmt.sh]: s/2014/2018

**Comment by m1093782566** [comment]
@shashidharatd 

Comments are addressed now. Please take a look again when you have a chance. Thanks!

**Comment by m1093782566** [comment]
@shashidharatd @shivramsrivastava 


`/retest` seems dose not work for `pull-poseidon-bazel` job.

**Comment by m1093782566** [comment]
In addition, close this PR and reopen would not trigger `pull-poseidon-bazel` job re-run once.

**Comment by BenTheElder** [comment]
[trigger plugin was re-enabled]
/test pull-poseidon-bazel

**Comment by BenTheElder** [comment]
ah this needed #33 as well
/test pull-poseidon-baze

**Pull request merged**